### ISSUE TITLE
Move Technical Direction Document to a more suitable place on the Website

### ISSUE THREAD

**Issue by ivargrimstad** [eclipse-ee4j/ee4j#28]
Currently published as a news item:

https://www.eclipse.org/ee4j/news/?date=2018-06-08

**Comment by ivargrimstad** [comment]
Project link from the website.

**Comment by waynebeaton** [comment]
I just pushed an update.

I've moved the "Technical Direction" document out of the news and promoted it into a [root document](https://www.eclipse.org/ee4j/direction.php). I've added a link to it in the right navigation and on the main project page.

While I was there, I added a root document for the [specification relationships document](https://www.eclipse.org/ee4j/specs.php). I haven't linked that to any of the navigation blocks yet.

For completeness, the sources are all in https://github.com/eclipse-ee4j/ee4j-website
